core/src/test/kotlin/org/evomaster/core/problem/rest/classifier/ModelMetricsWithTimeWindowTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.evomaster.core.problem.rest.classifier
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class ModelMetricsWithTimeWindowTest {
+
+
+    @Test
+    fun estimateModelMetrics() {
+
+        val delta = 0.0001
+        val ma = ModelMetricsWithTimeWindow(10) // buffer large enough to hold all
```

Comment:
besides this test, add back previous deleted test, where `val ma = ModelAccuracyWithTimeWindow(4)`.
that was done on purpose to verify the time window.
as rule of thumb, do not delete or modify existing tests, add new ones (of course if needed, changes/deletes can be discussed).
as wrote in a previous comment, I do not think the timewindow implementation is correct. to verify it, need a buffer that is smaller than the data you insert